lib/SILOptimizer/Utils/OptimizerStatsUtils.cpp

```diff
@@ -384,15 +387,20 @@ struct ModuleStat {
 struct InstCountVisitor : SILInstructionVisitor<InstCountVisitor> {
   int BlockCount = 0;
   int InstCount = 0;
+  const bool &NewFunc;
```

Comment:
It's not obvious what this means, can you document it?

Reply:
And why does it need to be a reference? Can it change during the visit?

Reply:
I think you focused on the `const` part, but my question was about why this needs to be a reference.